src/components/user/SignUpForm/index.js

```diff
@@ -6,23 +6,28 @@ import { View, Text } from 'react-native';
 import * as constraints from 'utils/constraints';
 import Input from 'components/common/Input';
 import Button from 'components/common/Button';
+import PickerScreen from 'components/common/PickerScreen';
 import translate from 'utils/i18n';
 import styles from './styles';
 import { whiteColor } from '../../../constants/styleConstants';
```

Comment:
Just
```suggestion
import { whiteColor } from 'constants/styleConstants';
```
will be fine